index.html

```diff
@@ -1,12 +1,233 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a class="navigation-logo" href="index.html">
+        <img class="page-header-logo" alt="Логотип Седона" src="">
```

Comment:
```suggestion
        <img class="page-header-logo" alt="Логотип Седона." src="">
```

в альтах ставим точки, для интонационной паузы, чтобы скрин-ридер сделал остановку - можешь в инете почитать по-подробнее зачем. Или, если интересно, я могу надиктовать ответ в телеге